pkg/applier/stackapplier.go

```diff
@@ -100,3 +105,17 @@ func (s *StackApplier) DeleteStack(ctx context.Context) error {
 
 // Health-check interface
 func (s *StackApplier) Healthy() error { return nil }
+
+func (*StackApplier) triggersApply(event fsnotify.Event) bool {
+	// always let the initial apply happen
+	if event == (fsnotify.Event{}) {
+		return true
+	}
+
+	// ignore chmods (3845479a0)
```

Comment:
Is `3845479a0` a commit of interest?

Reply:
That was the commit that initially introduced the filtering for chmods.